iam_openapi_spec.yml

```diff
@@ -932,6 +932,27 @@ paths:
         default:
           $ref: '#/components/responses/ErrorResponse'
 
+  /token:
+    post:
+      tags:
+        - User Authorization
+      summary: Generate a token
+      description: Generate a token for the given user credential (same as /organizations/{organization_id}/token at the moment. Might change in future)
+      operationId: getToken
+      responses:
```

Comment:
nit: We can also add header in the responses so that it will be visible in the openapi explorer for clients.

https://swagger.io/docs/specification/2-0/describing-responses/